server/src/main/java/org/elasticsearch/index/engine/EngineConfig.java

```diff
@@ -256,9 +256,9 @@ public Codec getCodec() {
     }
 
     /**
-     * @return the {@link CodecService}
+     * @return the {@link CodecSupplier}
      */
-    public CodecService getCodecService() {
+    public CodecSupplier getCodecService() {
```

Comment:
This could/should be renamed, but wanted to check with reviewers names were good (e.g. `CodecSupplier`) before doing it. Wdyt?

Reply:
yes, I think it should be renamed unless it touches a million places. Is it going to break the serverless build if renaming?

Reply:
I don't think so, but I'll try (add the label)